connector_ecommerce/models/account_payment_method.py

```diff
@@ -0,0 +1,43 @@
+# © 2011-2013 Akretion (Sébastien Beau)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+import re
+
+from odoo import api, models
+
+
+class AccountPaymentMethod(models.Model):
+    _inherit = "account.payment.method"
+
+    @api.model
+    def _get_payment_method_domain(self, payment_method):
```

Comment:
Method signature changed in https://github.com/odoo/odoo/commit/b4ac3dab7c0da521d829e5fffb267a735d6fe5bb#diff-50b738d9e0cb94e086b0b44558a90cd3badbcbb22c743efad97b27bbe23bf431R40